core/client/app/components/gh-select-native.js

```diff
@@ -21,7 +21,7 @@ export default Component.extend({
 
     actions: {
         change() {
-            let [selectEl] = this.$('select');
+            let selectEl = this.$('select')[0];
```

Comment:
You'll need to disable the `requireArrayDestructuring` jscs rule here for the linting to pass.